packages/p2p/src/utils/file-uploader.js

```diff
@@ -1,4 +1,4 @@
-import { localize } from '@deriv/translations';
+import { localize } from 'Components/i18next';
```

Comment:
would you please elaborate why we dont need the localize from translation package anymore?

Reply:
p2p has its own `localize` and `Localize` component